### DISCUSSION TITLE
How to convert from raw ULS data to .sqlite3 manually

### DISCUSSION THREAD

**Discussion by sambk** [open-afc-project/openafc#245] [Q&A]
Hi openafc team,
I'm trying to run SUT Test vector so I'm preparing databases from this https://github.com/Wi-FiTestSuite/6GHz-AFC/tree/main/ULS_database/FCC/weekly. 

After downloading the ULS raw data, I would like to convert it to .sqlite3 to be used. However, I don't see any instruction from openafc about this. Could anyone help to tell me how to properly run this conversion manually?

Thank you,
Pham Viet Sam

**Comment by sidmalli** [comment]
@sambk Any APIs in uls-downloader where we can input the .zip file URL and generate a .sqlite3 file? Please point towards the steps

**Comment by sidmalli** [comment]
@rkfteam Could you please give us any steps for this conversion

**Reply by mgelman-rkf** [reply]
@sidmalli, the download and conversion to sqlite3 is a multistep process and is automated for daily download.  There isn't really a manual process that we have packaged.  You can look at src/ratapi/ratapi/daily_uls_parse.py to see how the process works and uls/uls_service.py to see how it's called.
